internal/engine/allexperiments.go

```diff
@@ -122,6 +123,38 @@ var experimentsByName = map[string]func(*Session) *ExperimentBuilder{
 		}
 	},
 
+	"example_with_default_input": func(session *Session) *ExperimentBuilder {
+		return &ExperimentBuilder{
+			build: func(config interface{}) *Experiment {
+				return NewExperiment(session, example.NewExperimentMeasurer(
+					*config.(*example.Config), "example_with_default_input",
+				))
+			},
+			config: &example.Config{
+				Message:   "Good day from the example with input experiment!",
+				SleepTime: int64(time.Second),
+			},
+			interruptible: true,
+			inputPolicy:   InputOrStaticDefault,
+		}
+	},
+
+	"example_with_default_input_wrongly_configured": func(session *Session) *ExperimentBuilder {
+		return &ExperimentBuilder{
+			build: func(config interface{}) *Experiment {
+				return NewExperiment(session, example.NewExperimentMeasurer(
+					*config.(*example.Config), "example_with_default_input_wrongly_configured",
+				))
+			},
+			config: &example.Config{
+				Message:   "Good day from the example with input experiment!",
+				SleepTime: int64(time.Second),
+			},
+			interruptible: true,
+			inputPolicy:   InputOrStaticDefault,
+		}
+	},
+
```

Comment:
These new variations of the example experiment are useful to test `pkg/oonimkall` with a static default policy